haystack/utils/doc_store.py

```diff
@@ -119,17 +118,26 @@ def stop_service(document_store, delete_container=False):
         logger.warning("No support yet for auto stopping the service behind a %s", type(document_store))
 
 
-def launch_milvus(sleep=15, delete_existing=False):
-    """
-    Start a Milvus server via Docker
+def launch_milvus(sleep=15, delete_existing=False, timeout: Union[float, Tuple[float, float]] = 10.0):
+    """Start a Milvus server via Docker
+
+    :param sleep: How many seconds to wait after Milvus container is launched. Defaults to 15.
+    :type sleep: int
```

Comment:
No need to add the types here as we already have them in the type annotations in the methods signature